tests/agent_server/conftest.py

```diff
@@ -178,6 +178,14 @@ class BadeEventR(Resource):
 
         fields = ("key", "value", "purged")
 
+    @resource("test::BadEventsStatus", agent="agent", id_attribute="key")
+    class BadEventS(Resource):
+        """
+        A file on a filesystem
+        """
```

Comment:
This doesn't seem accurate, is it?

Reply:
True. That's a copy-paste issue. I will fix it